spec/lib/metasploit/concern/loader_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'spec_helper'
 
 RSpec.describe Metasploit::Concern::Loader do
+  include Shoulda::Matchers::ActiveModel
+  include Shoulda::Matchers::ActiveRecord
```

Comment:
Docs link for future travellers

https://github.com/thoughtbot/shoulda-matchers/tree/50085f03d9b5dc0ff8a412e9f62c9b4a5c4f3790#non-rails-projects